feature-libs/product-configurator/rulebased/components/attribute/types/multi-selection-bundle/configurator-attribute-multi-selection-bundle.component.ts

```diff
@@ -178,4 +180,40 @@ export class ConfiguratorAttributeMultiSelectionBundleComponent
       this.onHandleAttributeQuantity(eventObject.quantity);
     }
   }
+
+  isAnyValueSelected(attribute: Configurator.Attribute): boolean | undefined {
+    return attribute?.values?.some(
+      (value: Configurator.Value) => value?.selected
+    );
+  }
+
+  getSelectedValuesPrice(
+    attribute: Configurator.Attribute
+  ): number | undefined {
+    return attribute?.values
+      ?.filter((value: Configurator.Value) => value?.selected)
+      ?.reduce((total, value) => total + (value?.valuePrice?.value ?? 0), 0);
+  }
+
+  getSelectedValuesPriceTotal(
+    attribute: Configurator.Attribute
+  ): number | undefined {
+    return attribute?.values
+      ?.filter((value: Configurator.Value) => value?.selected)
+      ?.reduce(
+        (total, value) => total + (value?.valuePriceTotal?.value ?? 0),
+        0
+      );
+  }
```

Comment:
shouldn't we do this kind of calculation either in CPQ or at least in the normalizer? @AlexanderKuschner: What do you think?
considering angular change detection mechanism, its not good to do such calculations on the fly again and again.

Reply:
The methods have been removed.